_component_design/toggle.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Toggle
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: >
+  A toggle controls the presentation format for information on a control panel page.
+preview-image: preview-images/toggles.svg
+status: stable
+resource: true
+last-modified: 2019-03-13
+helix-ui-css: true
+helix-ui-javascript: true
+pagelink: https://rackerlabs.github.io/helix-ui/components/toogle/
```

Comment:
Remove this line; there's no toggle component yet.

Reply:
Done.